docs/proposals/mapper- provide-a- common- MQTT- mapper.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Mapper Provide A Common MQTT Mapper
```

Comment:
This description is a bit strange, how about using Provide A Common MQTT Mapper directly?

Reply:
Ok, I will take you advice.